bower.json

```diff
@@ -1,13 +1,13 @@
 {
-  "name": "adapt-mediaWithBenefits",
-  "version": "1.0.0",
+  "name": "adapt-mediaGallery",
+  "version": "2.0.0",
```

Comment:
version v1.0.0 hasn't been released so there's no particular need to bump this to v2.0.0